data_structures/quadTree/draw.py

```diff
@@ -0,0 +1,45 @@
+import turtle
+
+win = turtle.Screen()
```

Comment:
Please indent global code under `if __name__ == "__main__":` as discussed in CONTRIBUTING.md.